fedimint-server/src/consensus/mod.rs

```diff
@@ -647,15 +634,14 @@ impl FedimintConsensus {
             .collect()
             .await;
 
-        let mut items: Vec<ConsensusItem> = dbtx
-            .find_by_prefix(&ProposedTransactionKeyPrefix)
-            .await
-            .map(|res| {
-                let (_key, value) = res.expect("DB error");
-                ConsensusItem::Transaction(value)
-            })
-            .collect()
-            .await;
+        let mut items: Vec<ConsensusItem> = self
+            .tx_cache
+            .lock()
+            .unwrap()
```

Comment:
nit: prefer `expect` over `unwrap`

Reply:
I think `.lock()` is a special case that doesn't allow `expect`

Reply:
🤨

Reply:
No way. `Result` is a `Result`, has both `expect` and `unwrap`.